internal/catalog/internal/types/validators_test.go

```diff
@@ -373,26 +373,6 @@ func TestValidatePortName(t *testing.T) {
 	})
 }
 
-func TestValidateProtocol(t *testing.T) {
```

Comment:
Are these removed by accident?

Reply:
Yep, I added those a while back before I found out about `enumcover` 😄 the added annos should cover what they did